bugsnag-android-core/src/test/resources/error_serialization_0.json

```diff
@@ -0,0 +1,161 @@
+{
+  "metaData": {},
+  "severity": "error",
+  "severityReason": {
+    "type": "unhandledException"
+  },
+  "unhandled": true,
+  "incomplete": false,
+  "exceptions": [
+    {
+      "errorClass": "java.lang.RuntimeException",
+      "message": "whoops",
+      "type": "android",
+      "stacktrace": [
```

Comment:
Is this stacktrace guaranteed?  Is there a possibility this could change on different Java versions?

Reply:
For this particular test case the stacktrace should be guaranteed, as the object has constructed its own `StacktraceElement` rather than asking the JVM to generate one via `getStacktrace()`.